modules/logging-release-notes-5-8-10.adoc

```diff
@@ -0,0 +1,30 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-10_{context}"]
+= Logging 5.8.10
+This release includes link:https://access.redhat.com/errata/RHBA-2024:4629[OpenShift Logging Bug Fix Release 5.8.10] and link:https://access.redhat.com/errata/RHBA-2024:4665[OpenShift Logging Bug Fix Release 5.8.10].
+
+[id="logging-release-notes-5-8-10-known-issues"]
+== Known issues
+
+* Before this update, while enabling retention, the Loki Operator produced an invalid configuration. As a result, Loki did not start up properly. With this update, Loki pods work fine with the ability to set retention. (link:https://issues.redhat.com/browse/LOG-5821[LOG-5821])
```

Comment:
```suggestion
* Before this update, when enabling retention, the Loki Operator produced an invalid configuration. As a result, Loki did not start properly. With this update, Loki pods can set retention. (link:https://issues.redhat.com/browse/LOG-5821[LOG-5821])
```
Just "start" https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s 
The rest for minimalism